BSP/Simulator/Hardware/simulate.py

```diff
@@ -16,35 +15,6 @@ def update_buttons():
             button.config(relief=tk.RAISED)
     window.after(1, update_buttons)
 
-def update_timer():
```

Comment:
I would keep this function here, and have it call the function from the Timer.py file and also window.after()